### ISSUE TITLE
Tutorial helping create custom characteristics

### ISSUE THREAD

**Issue by isa3bel** [gregiOS/Playgrounds#1]
Hello,

I am using my MacBook as a peripheral, but it doesn't give off characteristics, and I'm trying to create two custom characteristics for read and write. Does this tutorial cover how to create custom characteristics https://itnext.io/swift-playgrounds-simulating-bluetooth-low-energy-cf5738c15a8c? 

Thanks!

**Comment by gregiOS** [comment]
Hello,
Thank you for your Interest in my project. 
Yes the tutorial shows how to setup peripheral with custom characteristics. 
In case of any issues feel free to ask her for a help.

**Comment by isa3bel** [comment]
Yes thank you! Will you please keep this issue open in case I want to ask for additional support? Thank you!

**Comment by isa3bel** [comment]
Hey @gregiOS, I was wondering where in my code currently I should add the custom characteristic. 
<img width="430" alt="image" src="https://user-images.githubusercontent.com/18249360/87100950-87114300-c202-11ea-9c2e-8670beb11ad1.png">
this is what my console looks like. It's showing 0 bytes, which means that the service is giving off no characteristics?  I'm not too sure how all of this works to be honest, but I provided you access to the repo if you wanna check out the code. specifically in the ViewController (https://github.com/MountLocks/NewMountApp/blob/master/ScannerApp/ViewController.swift)
Thanks for any help
